### ISSUE TITLE
[BUG] 'Dummy' object has no attribute 'downloaded'

### ISSUE THREAD

**Issue by FrenchGithubUser** [nathom/streamrip#763]
### Describe the bug

I'm trying to use the streamrip lib in a python script like so :

`
async def main():
    try:
        config = Config("streamrip_config.toml")
        deezer_client = DeezerClient(config=config)
        await deezer_client.login()
        print("Logged in:", deezer_client.logged_in)
        p = PendingAlbum(album_id, deezer_client, config, Dummy())
        resolved_album = await p.resolve()
        print(resolved_album.meta)  # Print metadata for verification
        await resolved_album.rip()
        print("Release downloaded successfully!")
        
    except Exception as e:
        print("An error occurred while downloading the release:")
        print(e)
 

if __name__ == "__main__":
    asyncio.run(main())
`

And getting this error :

`
'Dummy' object has no attribute 'downloaded'
Unclosed client session
client_session: <aiohttp.client.ClientSession object at 0x75bef26d0440>
Unclosed connector
connections: ['[(<aiohttp.client_proto.ResponseHandler object at 0x75bef2676450>, 19694.222413349), (<aiohttp.client_proto.ResponseHandler object at 0x75bef2676750>, 19694.304548539)]']
connector: <aiohttp.connector.TCPConnector object at 0x75bef26d0470>
`

### Command Used

```bash
python lib
```

### Debug Traceback

```text
done
```

### Config File

```toml
not needed
```

### Operating System

linux

### streamrip version

2.0.5

**Comment by Ilaprgg2** [comment]
that's not how it works.
`from streamrip.db import Dummy,  Database
...
db = Database(downloads=Dummy(), failed=Dummy())
...
 p = PendingAlbum(album_id, deezer_client, config, db)

`

**Comment by FrenchGithubUser** [comment]
Thanks for the correction !

I used what was in the [wiki](https://github.com/nathom/streamrip/wiki/Scripting-with-Streamrip-v2)

**Comment by Ilaprgg2** [comment]
yes. the information on the wiki is so minimal. I needed to review the source code to understand what it is
